source/grid/tria.cc

```diff
@@ -15672,10 +15833,14 @@ void Triangulation<dim, spacedim>::execute_coarsening_and_refinement()
   // Inform all listeners about beginning of refinement.
   signals.pre_refinement();
 
+  this->pack_data_serial();
+
   execute_coarsening();
 
   const DistortedCellList cells_with_distorted_children = execute_refinement();
 
+  this->unpack_data_serial();
```

Comment:
What is the purpose of `unpack_data_serial` here?

The function just updates the `local_cell_relations`, right? But you call `update_cell_relations()` already in line 15868.